src/main/resources/create_tables.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE teacher
```

Comment:
Firstly, all names of the tables should be in plural form(`teacher` -> `teachers`).

Reply:
It would be better to change all `VARCHAR` to `TEXT`.

Reply:
Try to execute the script in real DB in psql. I believe you have a few mistake in script.